evap/static/js/sortable_query_binding.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
to make those other changes possible. before i was getting raw DOM elements with get(), and now i can directly call sortable() on a jquery object. with this i also don't need to iterate over the two tables in the questionnaire index manually.